metaflow/util.py

```diff
@@ -9,7 +9,6 @@
 from itertools import takewhile
```

Comment:
this can be a different PR, cc @madhur-ob

Reply:
I don't think that is a change I did. What was this supposed to do?